generators/component/templates/component.ts

```diff
@@ -10,21 +11,62 @@ import * as Mes from "cmf.mes/src/mes";
 import i18n from "./i18n/<%= component.name %>.default"; 
 /** Angular */
 import * as ng from "@angular/core";
+//#endregion
+
+//#region Exports
+
+//#endregion
+
+//#region Constants
+
+//#endregion
 
 /**
- * Please provide a meaningful description of this component and how to use it
+ * ## Component name
+ * 
+ * Please provide a meaningful description of this component
+ * Try to answer these questions: 
+ * What is it?
+ * What does it do?
+ * Where to use it?
+ * How to use it?
+ * How it behaves with different sizes?
```

Comment:
The grammar is not quite right on this one, can you change this line to _"How does it behave with different sizes?"_